sdk/communication/communication-callingserver/src/callingServerClient.ts

```diff
@@ -824,4 +830,81 @@ export class CallingServerClient {
       span.end();
     }
   }
+
+    /**
+   * Deletes the content pointed to the uri passed as a parameter.
+   *
+   * * In Node.js, data returns ?
```

Comment:
don't we know what it returns? I see that in the Promise it should return `RestResponse` do we probably want that here.

Reply:
Fixed, sorry I missed this :)